### ISSUE TITLE
[Bug] Lightbeam causes browser to freeze

### ISSUE THREAD

**Issue by zero77** [mozilla/lightbeam-we#243]
I have a lot of entries in lightbeam and whenever I open the add on the browser starts to freeze and slow down to the point where it unusable.

**Comment by princiya** [comment]
There is a similar bug reported https://github.com/mozilla/lightbeam-we/issues/244#issuecomment-387569776

**Comment by zero77** [comment]
@princiya
Yes, it’s a bit similar but, I am not importing or exporting anything or changing Lightbeam in any way.

I have just been using Firefox normally and now I cannot open Lightbeam anymore without Firefox completely crashing.

**Comment by davidcraig** [comment]
Is it possible to perform a reset of the lightbeam data without opening firefox? Ie through the file system, mine racked up to 1696 sites and the reset button opened the modal, tried to click ok but it's still trying to render the graph, it looks like a scene from the matrix :)
